vtm/src/org/oscim/map/Animator2.java

```diff
@@ -105,6 +103,12 @@ public void animateFlingRotate(float angularVelocity, float pivotX, float pivotY
         }
     }
 
+    public void animateFlingScroll(float velocityX, float velocityY) {
+        animateFlingScroll(velocityX, velocityY,
+                Integer.MIN_VALUE, Integer.MAX_VALUE,
+                Integer.MIN_VALUE, Integer.MAX_VALUE);
+    }
```

Comment:
If scrolling very "hard", scroll reaches the border values, and it doesn't look very good if it suddenly stops, as in maps there shouldn't be any borders (except at north and south pole, where it you couln't go over anyway).

Reply:
If mean `Viewport` limits, maps need to can have borders.
Maps should show also specific limited countries / regions.

Reply:
Yes of course, but this isn't handled by animator, rather by the `Viewport.setMapLimit()`. 
Above only provides animator from suddenly stop during scroll without any reason.